examples/trace-elasticsearch7.js

```diff
@@ -18,6 +18,8 @@ const apm = require('../').start({ // elastic-apm-node
   logUncaughtExceptions: true
 })
 
+// Note that version *8* is installed by default. To use v7 you'll need to:
```

Comment:
nit: Might be better to say _version 7 is not installed by default_ instead of _Note that version *8* is installed by default_, since the later will inevitably be not true one day but the former will almost certainly be true into the future.

Reply:
Sounds good to me, thanks.

Note, if interested: For a while I'd had both v7 and v8 installed using an alias for v7 -- `"@elastic/elasticsearch7": "npm:@elastic/elasticsearch@7.x"`. However the default npm in node v8 doesn't support that alias syntax, so I dropped it.

Reply:
commit f8631c07